charts/esphome/1.3.0/questions.yaml

```
@@ -75,6 +75,30 @@ questions:
       default: "Etc/UTC"
       $ref:
         - "definitions/timezone"
+
+  - variable: PUID
+    group: "Configuration"
+    label: "PUID"
+    description: "The UserID of the user running the application and owning the files"
+    schema:
+      type: int
+      default: 568
```

Can this be `1880` - or are we changing the user inside the container as well ?

I'm not sure if this is needed. I'm only suggesting it to match what I needed to get the host path working

```suggestion
      default: 1880
```